superset/models/helpers.py

```diff
@@ -750,13 +750,30 @@ def columns(self) -> List[Any]:
     def get_fetch_values_predicate(self) -> List[Any]:
         raise NotImplementedError()
 
+    @property
+    def default_endpoint(self) -> sa.Text:
+        raise NotImplementedError()
+
+    @property
```

Comment:
i don't think we need type here since it's already defined in both datasources

Reply:
Oh I incluyded this because of the mypy error